micro/core/utils.py

```diff
@@ -0,0 +1,10 @@
+import os
+import sys
+
+
+def set_folder(path):
+    try:
+        os.makedirs(path, exist_ok=True)
+    except PermissionError:
```

Comment:
Here again

Reply:
solved in commit b2f08aba5dff0c10a7c217c824e0086e5472c060